Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Dockerfile for automated build of signal-cli
+#
+# Refer to the signal-cli github pages for detailed Usage.
+#
+FROM gradle:3.5-jdk7-alpine
+
+LABEL maintainer "Kayvan Sylvan <kayvansylvan@gmail.com>"
+
+USER root
```

Comment:
i think you misunderstood the point i was raising. i was suggesting to run the build as a non-root user and run the *install* as root only.

i do see that signal itself run as gradle, and that's fine. :) i just wonder if the build part of the install process should run as gradle as well.